src/styles/_shell.scss

```diff
@@ -11,6 +11,7 @@
   -ms-user-select: none;
   user-select: none;
   -webkit-tap-highlight-color: transparent;
+  cursor: none;
```

Comment:
1. there is `.fullscreen` class on the `.player` so I would use it.
2. per the ticket - "We need to monitor its movement and if it’s idle for more than 3 seconds - automatically hide it."

Reply:
So according to what was agreed with Ron, this will happen in any screen size and will disappear along with the bottom bar